client-service/src/main/java/com/telecominfraproject/wlan/client/controller/ClientController.java

```diff
@@ -42,429 +44,446 @@
 @RequestMapping(value="/api/client")
 public class ClientController {
 
-    private static final Logger LOG = LoggerFactory.getLogger(ClientController.class);
```

Comment:
why there is such a large diff file? it's very difficult to see the changes made. can you try to make a diff smaller?

Reply:
I will undo the formatting changes. I think we should have formatter settings sourced for the team to use to avoid this